WalletWasabi.Fluent/Styles/Themes/BaseDark.axaml

```diff
@@ -87,6 +87,7 @@
 
     <BoxShadows x:Key="OptimizePrivacyOptionBoxShadow1">0 6 16 0 #22000000</BoxShadows>
     <BoxShadows x:Key="OptimizePrivacyOptionBoxShadow2">0 9 23 0 #4A000000</BoxShadows>
+    <SolidColorBrush x:Key="ValidationFailed" Color="#C50500" />
```

Comment:
Let's try to use `SystemErrorTextColor`

Reply:
Done! Thank you! I've used TextControlForeground for the TextBlock QR Code content text. It will work with light and dark themes